src/components/Block/index.js

```diff
@@ -0,0 +1,118 @@
+
+
+const widthProps = separateLiteralToProps`
+	${({ width }) => (width === true ? 'width: 100%' : width && `width: ${(width)};`)}
+	${({ height }) => (height === true ? 'height: 100%' : `height: ${(height)};`)}
+	${({ minHeight }) => minHeight && `min-height: ${(minHeight)};`}
+	${({ minWidth }) => minWidth && `min-width: ${(minWidth)};`}
+	${({ maxHeight }) => maxHeight && `max-height: ${(maxHeight)};`}
+	${({ maxWidth }) => maxWidth && `max-width: ${(maxWidth)};`}
+`;
+
+const flexProps = separateLiteralToProps`
+	${({ flexDirection }) => flexDirection && `flex-direction: ${flexDirection};`}
+	${({ flex }) => flex && `flex: ${flex};`}
+	${({ flexWrap }) => flexWrap && `flex-wrap: ${flexWrap};`}
+	${({ justify }) => justify && `justify-content: ${justify};`}
+	${({ align }) => align && `align-items: ${align};`}
+	${({ color }) => color && `background-color: ${color};`}
+`;
+
+const paddingProps = separateLiteralToProps`
+	${({ pv }) => pv && `padding-vertical: ${(pv)};`}
+	${({ ph }) => ph && `padding-horizontal: ${(ph)};`}
+	${({ pt }) => pt && `padding-top: ${(pt)};`}
+	${({ pr }) => pr && `padding-right: ${(pr)};`}
+	${({ pb }) => pb && `padding-bottom: ${(pb)};`}
+	${({ pl }) => pl && `padding-left: ${(pl)};`}
+	${({ padding }) => padding && `padding: ${(padding)};`}
+	${({ p }) => p && `padding: ${(p)};`}
+`;
+/* Background: */
+
+const marginProps = separateLiteralToProps`
+	${({ mv }) => mv && `margin-vertical: ${(mv)};`}
+	${({ mh }) => mh && `margin-horizontal: ${(mh)};`}
+	${({ mt }) => mt && `margin-top: ${(mt)};`}
+	${({ mr }) => mr && `margin-right: ${(mr)};`}
+	${({ mb }) => mb && `margin-bottom: ${(mb)};`}
+	${({ ml }) => ml && `margin-left: ${(ml)};`}
+	${({ margin }) => margin && `margin: ${(margin)};`}
+`;
+
+const borderProps = separateLiteralToProps`
+	${({ corners }) => corners && `border-radius: ${corners};`}
+`;
+
+const transformProps = separateLiteralToProps`
+	${({ scale }) => scale && `transform: scale(${(scale)});`}
+	${({ rotate }) => rotate && `transform: rotate(${(rotate)}deg);`}
+`;
+const isExist = value => value !== undefined;
+
+const positionProps = separateLiteralToProps`
+	${({ zIndex }) => zIndex && `z-index: ${(zIndex)};`}
+	${({ top }) => isExist(top) && `top: ${(top)};`}
+	${({ left }) => isExist(left) && `left: ${(left)};`}
+	${({ right }) => isExist(right) && `right: ${(right)};`}
+	${({ bottom }) => isExist(bottom) && `bottom: ${(bottom)};`}
+`;
+
+const superProps = separateLiteralToProps`
+	${({ relative }) => relative && 'position: relative;'}
+	${({ absolute }) => absolute && 'position: absolute;'}
+	${({ centred }) => centred && 'justify-content: center; align-items: center;'}
+	${({ js }) => js && 'justify-content: flex-start;'}
+	${({ jc }) => jc && 'justify-content: center;'}
+	${({ je }) => je && 'justify-content: flex-end;'}
+	${({ jb }) => jb && 'justify-content: space-between;'}
+	${({ as }) => as && 'align-items: flex-start;'}
+	${({ ac }) => ac && 'align-items: center;'}
```

Comment:
For a user who does not know the project, such a name would be a terrible headache; choose more humane names
` <Block row ac>` i mean ac,as,je & e.t.c.